sonar-plugin-api/src/main/java/org/sonar/api/server/ws/impl/PartImpl.java

```diff
@@ -17,26 +17,28 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.api.issue;
+package org.sonar.api.server.ws.impl;
 
-import java.util.Arrays;
-import java.util.HashSet;
-import org.junit.Test;
-import org.sonar.api.batch.fs.internal.DefaultInputFile;
-import org.sonar.api.batch.fs.internal.TestInputFileBuilder;
-import org.sonar.api.batch.sensor.issue.internal.DefaultNoSonarFilter;
+import java.io.InputStream;
+import org.sonar.api.server.ws.Request;
 
-import static org.assertj.core.api.Assertions.assertThat;
+public class PartImpl implements Request.Part {
 
-public class NoSonarFilterTest {
```

Comment:
No idea why git detected that as move... 

I consider `NoSonarFilterTest` as a test of implementation class which should be moved to SonarQube codebase